handler.go

```diff
@@ -26,8 +26,9 @@ import (
 )
 
 type handler struct {
-	host  string
-	paths pathConfigSet
+	host      string
+	paths     pathConfigSet
```

Comment:
why do we need both parsed paths and parsedPathRules? Can't we merge them into one type?

Reply:
:+1: